README.md

```diff
@@ -3,6 +3,26 @@ provided by freedesktop.org (see http://freedesktop.org/wiki/Software/shared-mim
 
 [![Gem Version](https://img.shields.io/gem/v/mimemagic.svg)](http://rubygems.org/gems/mimemagic)
 
+*Warning:* If you are using a version of MimeMagic < 0.3.7, or version 0.4.0, you may well be in breach of the
+GPL due to a GPL licensed dependency that was bundled with this gem. You should update to a version >= 0.3.7 
+as soon as possible. See https://github.com/minad/mimemagic/issues/97 for details.
+
+Dependencies
+============
+
+You will require a copy of the Freedesktop.org shared-mime-info database to be available. If you're on Linux,
+it's probably available via your package manager, and will probably be in the location it's being looked for
+when the gem is installed.
+
+macOS users can install via Homebrew with `brew install shared-mime-info`.
```

Comment:
This does install the package, but the gem still fails to install, and supplying the path with `FREEDESKTOP_MIME_TYPES_PATH` is still required.  The way this is worded, that isn't clear.
```suggestion
macOS users can install via Homebrew with:

    brew install shared-mime-info
    export FREEDESKTOP_MIME_TYPES_PATH=/usr/local/Cellar/shared-mime-info/2.1/share/shared-mime-info/packages/freedesktop.org.xml
    
    # then one of gem install or bundle install
    gem install mimemagic
    bundle install
```

Reply:
Hmm, can you give it another try? A brew install, followed by just doing a gem install with a locally built version of the gem worked fine on my machine just now.

Reply:
@jellybob Sorry I wasn't clear, and my suggestion isn't that great either.  My comment was on the subtle association from the text layout in the readme as it is in this PR, which implies that you only need to set `FREEDESKTOP_MIME_TYPES_PATH` iff `you can't do either of those things`:

> If for whatever reason you can't do either of those things, you'll need to obtain a copy from the Internet. Set the environment variable `FREEDESKTOP_MIME_TYPES_PATH` before installing the gem in order to point the build at that location.

By juxtaposing the two sentences without any additional context it can easily be misunderstood.

The following works!
```
brew install shared-mime-info
export FREEDESKTOP_MIME_TYPES_PATH=/usr/local/Cellar/shared-mime-info/2.1/share/shared-mime-info/packages/freedesktop.org.xml
bundle install
```